lib/code_corps/stripe_service/stripe_connect_subscription.ex

```diff
@@ -112,7 +106,10 @@ defmodule CodeCorps.StripeService.StripeConnectSubscriptionService do
     |> Repo.insert
   end
 
-  defp to_create_attributes(%StripeConnectCard{} = card, %StripeConnectCustomer{} = customer, %StripeConnectPlan{} = plan, quantity) do
+  defp to_create_attributes(
+    %StripeConnectCard{} = card, %StripeConnectCustomer{} = customer,
+    %StripeConnectPlan{} = plan, %{"quantity" => quantity}
```

Comment:
Good catch. The validate call at the start of the process actually does all the pattern matching required.